package.json

```diff
@@ -11,6 +11,7 @@
     "author": "Unirep Team",
     "repository": "https://github.com/Unirep/Unirep",
     "scripts": {
+        "install": "git submodule init && git submodule update",
```

Comment:
Good to know that you could make it work with git submodules!
So if new changes were to be pushed to [circomlib](https://github.com/iden3/circomlib), this script is to fetch updates right?

Reply:
Sorta, by default when you clone a repo the submodule directories are empty. This script initializes the submodule and checks out the branch specified in the `.gitmodules` file.

The submodule state actually isn't persisted to the repo, it's a temporary setting in the local index. But if any changes are made to the submodule contents (e.g. they push a new commit to the circomlib master branch) then the submodule symlink will be marked as changed in any git diffs (so we won't accidentally silently commit submodule changes).

We'll almost certainly make our own fork of circomlib so we can pull in changes as we want, instead of getting them whenever they commit.

Really sucks that we can't point to a commit/tag. We could manually do it by making a long script that does like:

```sh
cd packages/circuits/circuits/circomlib
git checkout v2.0.5
```

But that seems annoying/complicated